.github/workflows/scheduled_mdlinkcheck.yml

```diff
@@ -0,0 +1,34 @@
+name: Check Markdown links
+
+on: 
+  schedule:
```

Comment:
why have a separate yaml file vs. triggering on the schedule in the existing file?

Reply:
I'm not sure if I can use more than one "on" block in a yaml file. For the scheduled checks, I want to create an issue if dead links are found, but I don't want to create an issue if dead links are found after a push.

Reply:
The current ci has multiple on: blocks (PR, merge to master, tag), so it should work

Why not create an issue on push? It's a "bug", so having it on the board to triage / fix seems like a good thing.

you can also check the action (like we do in dockerCI) and have different behavior.

two files is an option too - just think about developer experience and choose the better option - I'm OK either way

Reply:
I'll update the existing link checker to use multiple on blocks. I didn't want to create an issue on push because of the possibility that we may create a lot of replicas of the same issue if multiple people push to repository.

Reply:
I ended up keeping the yml files separate.